pkg/install/install.go

```diff
@@ -219,6 +223,21 @@ func retryingApply(path string) error {
 	return err
 }
 
+func reduceResources() error {
+	var err error
+	err = runCommand(exec.Command("kubectl", "set", "resources", "deployment", "activator", "--requests=cpu=150m,memory=30Mi", "--limits=cpu=500m,memory=300Mi", "-n", "knative-serving"))
```

Comment:
Kind node seems to share the host cpu and mem so given that the sum of the Serving deployments cpu requests (including Kourier) are ~=1cpu should we try set a resourceQuota on the serving ns? Another thing to explore is limit [Kind resource allocation.](https://github.com/kubernetes-sigs/kind/issues/1578#issuecomment-631140100) if we want to use on worker node (because of the quickstart). At the Serving side we use  `kind-worker-count=4` and setup Kind with `chainguard-dev/actions/setup-kind@main`.

Reply:
Thank you for this valuable suggestion. If I can limit resource usage from the kind point of view itself, then I can avoid hard reduction of deployment resources.